qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py

```diff
@@ -165,6 +165,14 @@ def _setup_help(self):
             raise Exception("No Component found.")
 
         filepath = self.qcomponent_file_path
+
+        imageStringPath = filepath.replace(".py", ".png")
+        imagefilename = os.path.basename(os.path.normpath(imageStringPath))
```

Comment:
Nice, you can also use pathlib for most of this too usually in terms of resolving path etc